File: auto-docs/executables/julia/privacy_false.jl
using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

data = [
  [
    "x" => [0, 2, 4], 
    "y" => [0, 4, 2], 
    "type" => "scatter"
  ]
]
response = Plotly.plot(data, ["filename" => "privacy-false", "world_readable" => false, "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/r/privacy_false.r
# Learn about API authentication here: {BASE_URL}/r/getting-started
# Find your api_key here: {BASE_URL}/settings/api

library(plotly)

py <- plotly(username='TestBot', key='r1neazxo9w')
data <- list(
  list(
    x = c(0, 2, 4), 
    y = c(0, 4, 2), 
    type = "scatter"
  )
)
response <- py$plotly(data, kwargs=list(filename="privacy-false", world_readable=FALSE, fileopt="overwrite"))
url <- response$url
